docs/Contributing/release.rst

```diff
@@ -41,7 +41,7 @@ These are the general steps to prepare and finalize a release:
       open a PR for this change to ``master``. This way, development builds from ``master`` will immediately
       use the version number of the next stable.
 
-#. If the release is a :strong:`release candidate` or a :strong:`stable release`, the release manager will
+#. If the release is a :strong:`release candidate` for a :strong:`stable release`, the release manager will
```

Comment:
Why this change? This wasn't a typo

Reply:
no that change is correct. "If the release is a release candidate **for** a stable release...", the original "or" wasn't correct.